maxim/tests/test_prompts.py

```
@@ -9,10 +9,7 @@
 # reading testConfig.json and setting the values
 
 # Get the directory where this test file is located
-test_dir = os.path.dirname(os.path.abspath(__file__))
-config_path = os.path.join(test_dir, "testConfig.json")
-
-with open(config_path) as f:
+with open(str(f"{os.getcwd()}/maxim/tests/testConfig.json")) as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)